bin/mail/2016-05-20-TEALS/generate-list.rb

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# This is a follow-up email to ../2016-05-04-TEALS, targeting a wider audience (de-duped against the old list).
+# Spec: https://docs.google.com/document/d/16KXH4E3M0WWhPUE25j6Sooa1CN1qtyjQ2sbHcbiNqJo/edit#heading=h.9hz1cfop5llo
+
+# These values are arbitrary, based on need, from a snapshot of the data.
+NUM_VOLUNTEERS_BY_STATE = {
+  'Washington' => 249,
+  'California' => 142,
+  'New York' => 111,
+  'Texas' => 92,
+  'Virginia' => 82,
+  'Arizona' => 72,
+  'Massachusetts' => 69,
+  'Georgia' => 48,
+  'Rhode Island' => 40,
+  'Oregon' => 20,
+  'Florida' => 18,
+  'North Dakota' => 17,
+  'North Carolina' => 16,
+  'Kentucky' => 15,
+  'Illinois' => 14,
+  'Indiana' => 14,
+  'Michigan' => 14,
+  'Minnesota' => 14,
+  'South Carolina' => 13,
+  'Louisiana' => 12,
+  'Colorado' => 10,
+  'Utah' => 9,
+  'Pennsylvania' => 8,
+  'District of Columbia' => 5
+}
+
+previously_contacted = Set.new
+OLD_TEMPLATE = '2016-05-05-teals-volunteers'
+old_email_query = %Q(
+  SELECT DISTINCT contact_email
```

Comment:
It seems redundant (though not harmful) to make `previously_contacted` a set (rather than a hash), yet dedupe in the query through `DISTINCT`.